apps/src/templates/MultipleSectionsAssigner.jsx

```diff
@@ -195,24 +195,13 @@ const MultipleSectionsAssigner = ({
 
   return (
     <AccessibleDialog onClose={onClose}>
-      <div
-        tabIndex="0"
-        className={classnames(
-          'uitest-confirm-assignment-dialog',
```

Comment:
@levadadenys noted this was in the file twice. When I looked at it, I couldn't see evidence it was being used (a different UI test uses this classname but with a different component)

Reply:
Awesome, thanks for deleting these two!